src/main/java/core/basesyntax/handler/BetConsoleHandler.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.handler;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class BetConsoleHandler implements Handler {
```

Comment:
let’s have only one handler that will read info about both entities from the console
example of user input `200 0.6 banana 20`

Reply:
done

Reply:
removed FruitHandler